Streamline Remix Getting Started Page (#7482)
getsentry/sentry-docs: src/platform-includes/getting-started-primer/javascript.remix.mdx

</Note>

> Under the hood, Remix SDK relies on our [React SDK](/platforms/javascript/guides/react/) on the frontend and [Node SDK](/platforms/node) on the backend, which makes all features available in those SDKs also available in this SDK.

Reviewer: ```suggestion
Sentry's Remix SDK relies on our [React SDK](/platforms/javascript/guides/react/) for the frontend and [Node SDK](/platforms/node) for the backend. That means all features available in those SDKs are also available in this SDK.
```
